gravitee-apim-repository/gravitee-apim-repository-api/src/main/java/io/gravitee/repository/management/api/PortalPageRepository.java

```diff
@@ -23,4 +23,9 @@
  */
 public interface PortalPageRepository extends CrudRepository<PortalPage, String> {
     List<PortalPage> findByIds(List<String> ids);
+
+    /**
+     * Returns pages with the expand field for the given ids.
+     */
+    List<PortalPage> findByIdsWithExpand(List<String> ids, List<String> expand);
```

Comment:
As discussed today it would be nice if we can add the expand enum in the repository model package. We can add an adapter in the query service to keep on valid values in the parameter and do the get value at repository implementation level.